warehouse/email.py

```diff
@@ -19,7 +19,7 @@
 
 
 @tasks.task(bind=True, ignore_result=True, acks_late=True)
-def send_email(task, request, body, recipients, subject):
+def send_email(task, request, body, subject, recipients=None):
```

Comment:
This doesn't actually make it keyword only right? To do that it'd need to be ``*, recipients=None)`` right?

Reply:
Thanks! You're right. I've added the `*, `